scripts/india_lgd/local_government_directory_districts/test_data/lgd_allDistrictofIndia_export.csv

```diff
@@ -613,8 +615,7 @@
 696,"SURYAPET",36,"TELANGANA",,0
 698,"VIKARABAD",36,"TELANGANA",,0
 693,"WANAPARTHY",36,"TELANGANA",,0
-522,"WARANGAL RURAL",36,"TELANGANA",9,540
-686,"WARANGAL URBAN",36,"TELANGANA",,0
+522,"WARANGAL",36,"TELANGANA",9,540
```

Comment:
Do we need to update the prod MCFs? http://google3/datacommons/import/mcf/prod_base_mcfs.textproto;l=279-285;rcl=407203675